packages/react-native/sdks/hermes-engine/hermes-utils.rb

```diff
@@ -202,9 +212,14 @@ def hermestag_file(react_native_path)
 end
 
 def release_tarball_url(version, build_type)
+    maven_repo_url = "https://repo1.maven.org/maven2"
+    if react_native_maven_local_path_envvar_defined()
+        maven_repo_url = "file://#{ENV[REACT_NATIVE_MAVEN_LOCAL_PATH]}"
+    end
+    namespace = react_native_maven_namespace()
```

Comment:
the local maven repository does not contains the hermes tarball for iOS. It only contains Android related stuff: the local maven repository is created by the `build_android` job and has no artifact for ios

Reply:
See comment above.

Reply:
Fixed